Source/WebKitLegacy/mac/History/WebHistoryItem.mm

```diff
@@ -369,12 +349,6 @@ - (NSPoint)scrollPoint
     return core(_private)->scrollPosition();
 }
 
-- (void)_visitedWithTitle:(NSString *)title
-{
-    core(_private)->setTitle(title);
-    _private->_lastVisitedTime = [NSDate timeIntervalSinceReferenceDate];
```

Comment:
I suggest with keep this method, rename it to `_visited` and still update `_lastVisitedTime`.

Reply:
> I suggest with keep this method, rename it to `_visited` and still update `_lastVisitedTime`.

OK, made that change